['Outputs do not match', 'Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmproscz7n1/solution_3.py", line 49, in <module>\n    print(Solution().maximumEnergy(energy = [5, 5, 5, 5, 5, 5, 5, 5, 5, 5], k = 1))\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmproscz7n1/solution_3.py", line 36, in maximumEnergy\n    current_value = energy[index] + table[queue[0]]\nIndexError: deque index out of range\n']